centreon/packages/ui/src/components/Form/FormActions.tsx

```diff
@@ -27,17 +27,14 @@ const FormActions = <TResource extends object>({
   variant,
   enableSubmitWhenNotDirty,
   isCancelButtonVisible = true,
-  disableSubmit = false
+  disableSubmit = undefined
```

Comment:
```suggestion
  disableSubmit
```
undefined is already its default value